packages/sanity/src/core/form/__telemetry__/form.telemetry.ts

```diff
@@ -0,0 +1,13 @@
+import {defineEvent} from '@sanity/telemetry'
+
+export const ExpandPortableTextInput = defineEvent({
+  version: 1,
+  name: 'Expand PTE',
```

Comment:
Think this should rather be "Portable Text Editor Expanded"? (cc @svirs)

Reply:
Cool, will wait for @svirs confirmation on this.

Reply:
The latest has `Portable Text Editor expanded/collapsed` which is fine 👍